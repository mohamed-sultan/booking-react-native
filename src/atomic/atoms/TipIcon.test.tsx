import React from 'react';
import renderer from 'react-test-renderer';
import TipIcon from './TipIcon'; 
// Mock react-native-vector-icons
jest.mock('react-native-vector-icons/Ionicons', () => {
  return ({ name, size }: any) => {
    return `Ionicon name=${name} size=${size}`;
  };
});

describe('TipIcon', () => {
  it('renders correctly with a given size', () => {
    const tree = renderer.create(<TipIcon size={24} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with a different size', () => {
    const tree = renderer.create(<TipIcon size={32} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
