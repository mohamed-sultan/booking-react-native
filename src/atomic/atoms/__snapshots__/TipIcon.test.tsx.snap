// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TipIcon renders correctly with a different size 1`] = `null`;

exports[`TipIcon renders correctly with a given size 1`] = `null`;
